parsers/filter/filter.spec.ts

```diff
@@ -92,4 +92,21 @@ describe('Filter', () => {
     expect(result.length).toEqual(1);
     return;
   });
+
+  it('Should work on non-string values', async () => {
+    const result = await filter(
+      (await filter(
+        seeds().tokens as Array<Token>,
+        { key: 'type', regex: { pattern: 'bitmap' } },
+        libs,
+      )) as Array<Token>,
+      { key: 'meta.dimension', regex: '2' },
+      libs,
+    );
```

Comment:
Following the last comment the test should pass without a filter on bitmap:
```suggestion
    const result = await filter(
      seeds().tokens as Array<Token>,
      { key: 'meta.dimension', regex: '2' },
      libs,
    );
```